aws/aws_eip.go

```diff
@@ -2,36 +2,45 @@ package aws
 
 import (
 	"github.com/jmcgill/formation/core"
-	//"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws"
+	// "github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/davecgh/go-spew/spew"
```

Comment:
This doesn't look like it is used anymore - which I think will cause a compile error! Can you confirm this still compiles?